src/domain/platform/admin/templates/CommunityGuidelines/CommunityGuidelinesTemplateForm.tsx

```diff
@@ -69,7 +74,7 @@ const CommunityGuidelinesTemplateForm = ({
       validateOnMount
       onSubmit={onSubmit}
     >
-      {({ values }) => (
+      {({ values, setFieldValue }) => (
```

Comment:
setFieldValue is passed to `FormikReferenceSegment` to be able to add references to the array  (without calling to the server, only in the Formik state